Dockerfile

```diff
@@ -10,6 +10,8 @@ RUN sudo apk add python3
 RUN sudo ln -s /usr/bin/python3 /usr/bin/python
 RUN sudo apk add g++
 RUN sudo apk add make
+# We also need bash to build JaneStreet's base
+RUN sudo apk add bash
```

Comment:
```suggestion
```
You probably want to remove this part for now (see https://github.com/CatalaLang/catala/pull/478/commits/0c6ae736cb98c2a0a4742ee5c4e88a01459933db and #479) to avoid rebuilding the CI containers.